test/unit/views/ViewSubmissionsViewSpec.scala

```diff
@@ -140,17 +140,17 @@ class ViewSubmissionsViewSpec extends ViewSpec with Injector {
       val secondDataRowElements = pageWithData.selectFirst(".govuk-table__body .govuk-table__row:nth-child(2)")
       val thirdDataRowElements = pageWithData.selectFirst(".govuk-table__body .govuk-table__row:nth-child(3)")
 
-      firstDataRowElements.selectFirst(".ucr").text() mustBe validMucr
+      firstDataRowElements.selectFirst(".ucr").text() mustBe s"$validMucr ${messages("submissions.hidden.text", validMucr)}"
```

Comment:
It might be worth having a test with multiple UCRs. So you can test the `mkString` method.